### DISCUSSION TITLE
Why not distribute the framwork/library as a NPM Module?

### DISCUSSION THREAD

**Discussion by SIXS1X** [adityash4rma/flaregram#11] [Q&A] [answered]
Why not distribute the framework as an NPM Module? and do we have to git clone and setup it manually?

**Comment by adityash4rma** [comment, marked as answer]
Hey @SIXS1X,
I didn't distribute the framework as an NPM Module as, i think distributing it as an NPM Module will make the project slower and so I **currently** prefer sharing it as a git repository.

I will make the repository a wrangler template, so to make everything automatic and reducing the number of commands we use, when setting up the project!

Stay Tuned! 😉
